src/device_trezor/device_trezor.cpp

```diff
@@ -0,0 +1,354 @@
+//
+// Created by Dusan Klinec on 01/08/2018.
+//
+
+#include "device_trezor.hpp"
+
+namespace hw {
+namespace trezor {
+
+#if WITH_DEVICE_TREZOR
+
+#undef MONERO_DEFAULT_LOG_CATEGORY
+#define MONERO_DEFAULT_LOG_CATEGORY "device.trezor"
+
+#define HW_TREZOR_NAME "Trezor"
+#define HW_TREZOR_NAME_LITE "TrezorLite"
+
+    static device_trezor *trezor_device = nullptr;
+
+#if WITH_DEVICE_TREZOR_LITE
+    static device_trezor_lite *trezor_device_lite = nullptr;
+#endif
+
+    static device_trezor *ensure_trezor_device(){
+      if (!trezor_device) {
+        trezor_device = new device_trezor();
+        trezor_device->set_name(HW_TREZOR_NAME);
+      }
+      return trezor_device;
+    }
+
+#if WITH_DEVICE_TREZOR_LITE
+    static device_trezor_lite *ensure_trezor_device_lite(){
+      if (!trezor_device_lite) {
+        trezor_device_lite = new device_trezor_lite();
+        trezor_device_lite->set_name(HW_TREZOR_NAME);
+      }
+      return trezor_device_lite;
+    }
+#endif
+
+    void register_all(std::map<std::string, std::unique_ptr<device>> &registry) {
+      registry.insert(std::make_pair(HW_TREZOR_NAME, std::unique_ptr<device>(ensure_trezor_device())));
+#if WITH_DEVICE_TREZOR_LITE
+      registry.insert(std::make_pair(HW_TREZOR_NAME_LITE, std::unique_ptr<device>(ensure_trezor_device_lite())));
```

Comment:
ensure_trezor_device_lite uses HW_TREZOR_NAME, whereas this uses HW_TREZOR_NAME_LITE - intended ?

Reply:
It should be `HW_TREZOR_NAME_LITE`, now it is OK. Thanks